.github/actions/install_ubuntu_deps/action.yml

```diff
@@ -0,0 +1,71 @@
+name: install_ubuntu_deps
+runs:
+  using: composite
+  steps:
+  - name: install cmake
+    # OpenEXR requires CMake 3.12. We also download CMake 3.20, which
+    # we'll use later for the JS build only.
+    run: |
+      echo $(date +%F) > ./date
+      echo $(git ls-remote https://github.com/facebookresearch/habitat-lab.git HEAD | awk '{ print $1}') > ./hablab_sha
+      cat ./hablab_sha
+      wget https://cmake.org/files/v3.12/cmake-3.12.4-Linux-x86_64.sh
+      sudo mkdir /opt/cmake312
+      sudo sh ./cmake-3.12.4-Linux-x86_64.sh --prefix=/opt/cmake312 --skip-license
+      sudo ln -s /opt/cmake312/bin/cmake /usr/local/bin/cmake
+      sudo ln -s /opt/cmake312/bin/ctest /usr/local/bin/ctest
```

Comment:
I would simplify this with `sudo apt install -y cmake`.

Reply:
Can try this and see if it works. :+1: